src/main/java/com/crowdin/client/translationmemory/model/TranslationMemory.java

```diff
@@ -14,7 +14,6 @@ public class TranslationMemory {
     private String name;
     private List<String> languageIds;
     private Integer segmentsCount;
-    private Long defaultProjectId;
-    private List<Long> projectIds;
```

Comment:
This `projectIds` property needs to be preserved

The API response contains both `projectIds` and `defaultProjectIds`, only the `defaultProjectId` should be removed since it's deprecating.

Reply:
Thank you for the feedback, guess now it's done, check, please

Reply:
@mitrofmep looks good to me, thank you!